pkg/k8s/endpointsynchronizer/cep.go

```diff
@@ -38,7 +39,9 @@ import (
 // of CiliumEndpoint resources.
 // TODO - see whether folding the global variables below into this function
 // is cleaner.
-type EndpointSynchronizer struct{}
+type EndpointSynchronizer struct {
+	Allocator *cache.IdentityAllocatorManager
```

Comment:
Where is this initialized? Can it be done so from this package, therefore providing better reassurance that it's always non-nil?

Reply:
It's currently done as part of the call to `endpointmanager.NewEndpointManager` in the daemon bootstrapping logic:

https://github.com/cilium/cilium/blob/b3e331892c7d90710a35635295bf1154434c1a69/daemon/daemon.go#L360-L362